internal/pkg/providers/aws/security_group_updator.go

```diff
@@ -0,0 +1,98 @@
+package aws
```

Comment:
@fahad-rauf Please change class name to `securitygroup_updater.go`. Notice `updater` instead of `updator`.

Also had a discussion with @waseem-h about this and he mentioned that `_` is only used when absolutely necessary. And based on that I think it should be `securtiygroup_updater.go` and not `security_group_updater.go`